### PR TITLE
Changed create files

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/FileService.Business/Commands/File/CreateFilesCommand.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Models.Project;
+using LT.DigitalOffice.Models.Broker.Enums;
+using LT.DigitalOffice.ProjectService.Broker.Requests.Interfaces;
+using Microsoft.AspNetCore.Http;
+
+namespace LT.DigitalOffice.FileService.Business.Commands.File
+{
+  public class CreateFilesCommand : ICreateFileCommand
```
Comment (SolovyovDmitry) [inline src/FileService.Business/Commands/File/CreateFilesCommand.cs]: different names of class and interface

File: src/FileService.Business/Commands/File/CreateFilesCommand.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      _mapper = mapper;
+      _accessValidator = accessValidator;
+      _projectService = projectService;
+      _httpContextAccessor = httpContextAccessor;
+      _publish = publish;
+    }
+
+    public async Task<OperationResultResponse<List<Guid>>> ExecuteAsync(Guid entityId, FileAccessType access, IFormFileCollection uploadedFiles)
+    {
+      List<ProjectUserData> users = await _projectService.GetProjectUsersAsync(new List<Guid>() { _httpContextAccessor.HttpContext.GetUserId() }, new());
```
Comment (MariyaSin) [inline src/FileService.Business/Commands/File/CreateFilesCommand.cs]: you do not use List errors
In this place no need create the list

File: src/FileService.Business/Commands/File/CreateFilesCommand.cs [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemoveProjects)
+        && !(users.Any() && users.FirstOrDefault().ProjectId == entityId && users.FirstOrDefault().ProjectUserRole == ProjectUserRoleType.Manager))
```
Comment (MariyaSin) [inline src/FileService.Business/Commands/File/CreateFilesCommand.cs]: change the conditions
AddEditRemoveProjects will add files extremely rarely

Comment (MariyaSin) [reply src/FileService.Business/Commands/File/CreateFilesCommand.cs]: Move FirstOrDefault to 52 line

File: src/FileService.Broker/Publishes/Publish.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  public class Publish : IPublish
+  {
+    private readonly IBus _bus;
+
+    public Publish(IBus bus)
+    {
+      _bus = bus;
+    }
+
+    public async Task CreateFilesAsync(Guid entityId, FileAccessType access, List<Guid> filesIds)
```
Comment (MariyaSin) [inline src/FileService.Broker/Publishes/Publish.cs]: remove async\await and return Task

Comment (VLiamin) [reply src/FileService.Broker/Publishes/Publish.cs]: done

File: src/FileService.Business/Commands/File/CreateFilesCommand.cs (hunk 2 above)
Comment (VLiamin) [reply src/FileService.Business/Commands/File/CreateFilesCommand.cs]: done

File: src/FileService.Business/Commands/File/CreateFilesCommand.cs (hunk 1 above)
Comment (VLiamin) [reply src/FileService.Business/Commands/File/CreateFilesCommand.cs]: done

File: src/FileService.Business/Commands/File/CreateFilesCommand.cs (hunk 3 above)
Comment (VLiamin) [reply src/FileService.Business/Commands/File/CreateFilesCommand.cs]: done

**Review by MariyaSin — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**